packages/material-ui/src/Table/index.d.ts

```diff
@@ -1,2 +1,4 @@
 export { default } from './Table';
 export * from './Table';
+export { default as TableContext } from './TableContext';
+export { default as Tablelvl2Context } from './Tablelvl2Context';
```

Comment:
Good question. I did this primarily so that I could squash the import statement into a single line and get the intellisense in my editor. But better start with private and see if people actually need the context.